torch/csrc/jit/codegen/cuda/root_domain_map.cpp

```diff
@@ -554,13 +558,22 @@ std::unordered_map<IterDomain*, IterDomain*> ComputeAtRootDomainMap::map(
     if (id_map.find(from_id) != id_map.end()) {
       continue;
     }
-    // Matching ID not found. It's an error unless from_id is a new
-    // broadcast of a consumer domain; or from_id is a window axis of
-    // a consumer domain. Note that reduction domains are removed from
-    // the producer root domain.
+    // Matching ID not found. It's an error unless the following three cases:
+    // 1. from_id is a new broadcast of a consumer domain; or
+    // 2. from_id is a window axis of a consumer domain; or
+    // 3. from_id is a split vector domain
+    // Note that reduction domains are removed from the producer root domain.
+    std::cout << "split_vector_domains_:" << std::endl;
+    for (auto i : split_vector_domains_) {
+      std::cout << "td = " << i.td() << ", id = " << i.id() << std::endl;
+    }
     if (!producer_to_consumer &&
-        (new_broadcast_domains_.find(DomainKey(from_td, from_id)) !=
+        (
+          new_broadcast_domains_.find(DomainKey(from_td, from_id)) !=
              new_broadcast_domains_.end() ||
+        //  from_id->getIterType() == IterType::SplitVector ||
```

Comment:
This makes sense and should not be commented out.

Reply:
Which line makes sense?
```C++
from_id->getIterType() == IterType::SplitVector ||
```
or
```C++
split_vector_domains_.find(DomainKey(from_td, from_id)) != split_vector_domains_.end() ||
```

Reply:
I think just `from_id->getIterType() == IterType::SplitVector` should be sufficient.